be/src/exec/es/es_scroll_parser.cpp

```diff
@@ -61,6 +61,15 @@ std::string json_value_to_string(const rapidjson::Value& value) {
     return scratch_buffer.GetString();
 }
 
+#define RETURN_NULL_IF_STR_EMPTY(col, column)                                          \
+    do {                                                                               \
+        const std::string& null_str = json_value_to_string(col);                       \
+        if ((null_str.length() <= 0 || "\"\"" == null_str) && column->is_nullable()) { \
+            _append_null(column);                                                      \
```

Comment:
Use `column->append_default()` instead.

I think you can delete `_append_null()` function, because it's only do `append_default()`